traefik/templates/_podtemplate.tpl

```diff
@@ -448,6 +448,7 @@
           - "--experimental.plugins.{{ $pluginName }}.moduleName={{ $plugin.moduleName }}"
           - "--experimental.plugins.{{ $pluginName }}.version={{ $plugin.version }}"
           {{- end }}
+          - "--experimental.abortonpluginfailure={{ .Values.experimental.abortOnPluginFailure }}"
```

Comment:
The chart doesn't support only the latest traefik version. You may need to add a test on traefik version here and update tests accordingly.

Reply:
Yes you're right, I made a change to make it work. We'll wait for the traefik version and I'll remove the “appVersion” from the tests.